### PR TITLE
Feat: About us page

### PR DESCRIPTION
**Pull request by MiraMC149**
this is the about us page but without the modal and I'm not sure if I added it to the router correctly
Screenshot (1233)
Screenshot (1234)
Screenshot (1235)
Screenshot (1236)
Screenshot (1237)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,13 +2,20 @@ import React from "react";
...
+import { BrowserRouter as Router, Switch,Route } from "react-router-dom";
+import { useHistory } from "react-router-dom";
+import About from "./pages/About/About";
+
 
 function App() {
   return (
     <Router>
       <Navbar />
+      <About />
```
Comment (osamaakb) [inline src/App.js:14]: you don't need this one just the one in the switch

File: src/components/TeamMemberCard/Member1.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Member1Image from "./../../images/Member1Image";
+function Member1() {
```
Comment (osamaakb) [inline src/components/TeamMemberCard/Member1.jsx:3]: replace the names in real members names like `MiraPhoto`

**Review by osamaakb — CHANGES_REQUESTED** [review]
Good job @MiraMC149

File: src/pages/About/About.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import Footer from "../../components/Footer/Footer";
+import OurValues from "../../components/OurValues/OurValues";
+import OurMission from "../../components/OurMission/OurMission";
+import ChangeTheWayYouShop from "../../components/ChangeTheWayYouShop/ChangeTheWayYouShop";
+import TeamMemberCard from "../../components/TeamMemberCard/TeamMemberCard";
+
+function AboutUs() {
+  return (
+    <div className="bg-mainbg">
+      <Navbar />
```
Comment (osamaakb) [inline src/pages/About/About.jsx:12]: why the Navbar and Footer are here if they are in the `app.js` already?

File: src/App.js [hunk 4]
```diff
@@ -4,14 +4,21 @@ import Navbar from "./components/Navbar/Navbar";
...
+import About from "./pages/About/About";
+import { BrowserRouter as Router, Switch,Route } from "react-router-dom";
+
 
 function App() {
 
   return (
     <Router>
       <Navbar />
       <Footer />
```
Comment (osamaakb) [inline src/App.js:16]: the footer should be under everything that is why it is called `Footer` (:

**Review by osamaakb — CHANGES_REQUESTED** [review]
Please pay more attention to the code and the output before submitting,

the `keep in touch` button should be under the team members and it should have some text maybe above it says `if you want more information about us` and then the button or something similar to make it more friendly
image

**Comment by MiraMC149** [comment]
ezgif com-gif-maker
This is a preview of the modal

**Review by osamaakb — APPROVED** [review]

**Pull request merged**